Gems/Terrain/Code/Source/Components/TerrainPhysicsColliderComponent.cpp

```diff
@@ -421,8 +422,14 @@ namespace Terrain
             updateHeightsMaterialsCallback(row, column, point);
         };
 
-        size_t numPointsX = AZStd::min(aznumeric_cast<size_t>(contractedAlignedEndGridPoint.GetX() - contractedAlignedStartGridPoint.GetX() + 1), m_heightfieldRegion.m_numPointsX);
-        size_t numPointsY = AZStd::min(aznumeric_cast<size_t>(contractedAlignedEndGridPoint.GetY() - contractedAlignedStartGridPoint.GetY() + 1), m_heightfieldRegion.m_numPointsY);
+        const size_t numPointsX =
+            AZStd::min(
+                aznumeric_cast<size_t>(contractedAlignedEndGridPoint.GetX() - contractedAlignedStartGridPoint.GetX() + 1),
```

Comment:
nit: if you used the same trick as above (calculate `contractedAlignedEndGridPoint - contractedAlignedStartGridPoint + 1` first), you could cut theses lines down and avoid the wonky line breaks.